commands/bake.go

```diff
@@ -282,10 +439,15 @@ func bakeCmd(dockerCli command.Cli, rootOpts *rootOptions) *cobra.Command {
 	flags.StringArrayVarP(&options.files, "file", "f", []string{}, "Build definition file")
 	flags.BoolVar(&options.exportLoad, "load", false, `Shorthand for "--set=*.output=type=docker"`)
 	flags.BoolVar(&options.printOnly, "print", false, "Print the options without building")
+	flags.BoolVar(&options.listTargets, "list-targets", false, "List available targets")
+	flags.BoolVar(&options.listVars, "list-variables", false, "List defined variables")
```

Comment:
Should we open another PR for these flags? I don't think this is the same use case as `call` related to https://github.com/docker/buildx/issues/1072

Also I think it should be a single flag `--list` defaulting to `targets` as value but can also be `variables`? I'm looking at https://github.com/casey/just#command-line-options

Reply:
Maybe we could consider using existing `--print` flag so we could pass `--print=targets` or `--print=variables` and `--print` would default to `--print=definition`?

Reply:
Unless we have a clear use case, I would be keen not to overcomplicate things at this point. We can always make additional updates in the future

Reply:
`list-targets` and `list-variables` have been marked as experimental in the meantime.